src/main/java/io/strimzi/kafka/metrics/KafkaMetricsCollector.java

```diff
@@ -110,7 +110,8 @@ String metricName(MetricName metricName) {
     }
 
     static MetricFamilySamples convert(String name, String help, KafkaMetric metric, Map<String, String> labels) {
-        Object value = metric.metricValue();
+        Object valueObj = metric.metricValue();
+        final double value;
```

Comment:
We don't use the `final` modifier on other local variables throughout the code base. Let's keep it consistent.

Reply:
`final` removed